Libraries/src/Amazon.Lambda.Annotations.SourceGenerator/ProjectFileHandler.cs

```diff
@@ -0,0 +1,47 @@
+﻿using Amazon.Lambda.Annotations.SourceGenerator.FileIO;
+using System;
+using System.Xml;
+
+namespace Amazon.Lambda.Annotations.SourceGenerator
+{
+    /// <summary>
+    /// Utilities for working with project (.csproj) files, such as allowing projects
+    /// to opt out of the CloudFormation description modification.
+    /// </summary>
+    public class ProjectFileHandler
+    {
+        private const string OptOutNodeXpath = "//PropertyGroup/AWSSuppressCloudFormationDescriptionModification";
+
+        /// <summary>
+        /// Determines if the project has opted out of modifying the CloudFormation template descriptio
```

Comment:
nit: you are missing an "n" at the end of "descriptio". I'd prefer if the entire description would change to something like "Determines if the project has opted out of providing telemetry to AWS"

Reply:
Fixed in amended commit.
>         /// Determines if the project has opted out of any Lambda Annotations telemetry